pyproject.toml

```diff
@@ -1,5 +1,45 @@
+[build-system]
+requires = ["setuptools >=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "keras"
+authors = [
+    {name = "Keras team", email = "keras-users@googlegroups.com"},
+]
+description = "Multi-backend Keras"
+readme = "README.md"
+requires-python = ">=3.9"
+license = {text = "Apache License 2.0"}
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3 :: Only",
+    "Operating System :: Unix",
+    "Operating System :: MacOS",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Software Development",
+]
+dynamic = ["version", "dependencies"]
+
+[project.urls]
+Home = "https://keras.io/"
+Repository = "https://github.com/keras-team/keras"
+
+[tool.setuptools.dynamic]
+version = {attr = "keras.src.version.__version__"}
+dependencies = {file = "requirements-common.txt"}
```

Comment:
So, this is not the right set

Reply:
This can be restored with the list from setup.py. But I thought sourcing this file would avoid duplication, as the former list is a subset of requirements-common.txt. Nevertheless, as documented, the user must meet the minimal install with `pip install -r requirements.txt`, which cannot be done with this setup due to the complexity of the requirements file. I've added a comment to re-iterate this detail.

Reply:
We only want to install as few deps as possible on the user side, so we insert the explicit list here